documentatie/gebruikersdocumentatie/instructies-invoerder.md

```diff
@@ -2,20 +2,34 @@
 
 Welkom bij Abacus! Deze tool is ontwikkeld om de verkiezingsuitslagen per stembureau na te rekenen en te controleren op fouten en afwijkende uitslagen. In dit document leggen we uit hoe je Abacus gebruikt als invoerder.
 
-Na het tellen van de stemmen wordt voor elk stembureau een proces-verbaal ingevuld. De cijfers op dit proces-verbaal neem je over in Abacus. Vervolgens controleert de applicatie of de cijfers kloppen, en als alles klopt kun je de resultaten opslaan. Als er fouten of waarschuwingen zijn, volg je de instructies op of overleg je het met je coördinator.
+Na het tellen van de stemmen wordt voor elk stembureau een proces-verbaal ingevuld. De cijfers op dit proces-verbaal neem je over in Abacus. Vervolgens controleert de applicatie of de cijfers kloppen met het proces-verbaal. Ook wordt gecheckt of er bijzondere afwijkingen zijn, bijvoorbeeld als het aantal ongeldige stemmen hoog is. Als er fouten of waarschuwingen zijn, volg je de instructies op of overleg je het met je coördinator.
+
+Elk stembureau wordt twee keer ingevoerd door twee verschillende invoerders. Na de tweede invoer controleert Abacus of de eerste en tweede invoer overeenkomen. Als alles klopt is de invoer van het betreffende stembureau compleet. Ook hier geldt dat je bij fouten of waarschuwingen de instructies volgt of overlegt met je coördinator.
```

Comment:
```suggestion
Elk stembureau wordt twee keer ingevoerd (door twee verschillende invoerders). De tweede invoerder krijgt een waarschuwing als diens invoer afwijkt van de eerste invoerder. Ook hier geldt dat je de instructies volgt of overlegt met je coördinator. Na de tweede invoer controleert Abacus of de eerste en tweede invoer overeenkomen. Als alles klopt is de invoer van het betreffende stembureau compleet. 
```

- haakjes toegevoegd, omdat het anders leest als twee invoerders voeren elk twee keer in
- stukje over fouten en waarschuwingen verplaatst omdat dat specifiek is voor de tweede invoerder